### DISCUSSION TITLE
Removing darkening for specific websites

### DISCUSSION THREAD

**Discussion by vlkvch** [bromite/bromite#1551] [Ideas]
Some websites have a native dark mode. That is why it would be a great feature to disable darkening on different websites.

**Comment by uazo** [comment]
websites can already manage through css `prefers-color-scheme` directive or with meta `color-scheme`
(ref https://web.dev/color-scheme).

**Reply by vlkvch** [reply]
Thanks for the answer.

**Comment by ghost** [comment]
> Some websites have a native dark mode. That is why it would be a great feature to disable darkening on different websites.

this feature has been added update to the latest release
